src/ralph/utils.py

```diff
@@ -84,3 +84,16 @@ def now():
     """Return the current UTC time in ISO format"""
 
     return datetime.datetime.now(tz=datetime.timezone.utc).isoformat()
+
+
+def nested_get(dictionary, keys):
+    """Get the nested dict value by keys array."""
+
+    if keys is None:
+        return None
+    for key in keys:
+        try:
+            dictionary = dictionary[key]
+        except (KeyError, TypeError):
+            return None
+    return dictionary
```

Comment:
What about using a more concise functional programing implementation?

```python
import operator
from functools import reduce

def get_dict_value_from_path(dict_: dict, path: str, sep: str = "."):
    return reduce(operator.getitem, path.split(sep), dict_)
```

Reply:
This is beautiful ! Thanks)
It is a bit slower https://pastebin.com/hG0F0bab (~18%) but much more readable.
Will update this